voctopublish/voctopublish.py

```diff
@@ -139,6 +139,23 @@ def publish(self):
         else:
             logging.debug("no youtube :(")
 
+        if self.ticket.shell_commands and self.ticket.master:
+            if self.config.get('general', {}).get('enable_shell_commands', False):
```

Comment:
```suggestion
            if self.config['general'].get('enable_shell_commands', False):
```

Configparsers own `.get()` does not work like `dict.get()`.